server/db/models/userProducts.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require("sequelize");
+const db = require("../db");
+
+const userProducts = db.define("userProducts", {
```

Comment:
The variable storing the model needs to be capitalized and singular. Also, the name of model in double quotes needs to be singular. Postgres makes it plural.